Makefile

```diff
@@ -15,6 +15,10 @@ venv:
 	venv/bin/pip install tox
 	venv/bin/pip install -e .
 
+test-with-build:
```

Comment:
Following the `test-unit` naming, maybe this could be `test-integration`? Otherwise this looks great :+1:

Reply:
Ack